packages/template-retail-react-app/app/components/product-tile/index.jsx

```diff
@@ -82,7 +83,7 @@ const ProductTile = (props) => {
     //TODO variants needs to be filter according to selectedAttribute value
     const variants = product?.variants
 
-    const priceData = getPriceData({...product, variants})
+    const priceData = useMemo(() => getPriceData({...product, variants}), [product, variants])
```

Comment:
Question - Do we need variants as a dependency along with product to memorize as variants is a property with in product ?

Reply:
The `variants` is there because the TODO comment made me to anticipate `variants` to be updated in a later PR. And I don't know what it will look like later.

Reply:
@vmarta Can you try to  update your branch with the feature branch? I believe we have some new code merged in that may look different from that you have on your current branch for the variant TODO above